### PR TITLE
SendFaultyMessages

### PR DESCRIPTION
**Pull request by Scretch9**
- adds three new API functions and buttons to the UI to send faulty messages

### CODE REVIEW

File: src/Point/Point.cs [hunk 1]
```diff
@@ -101,6 +103,62 @@ public async Task SendSciMessage(SciMessage message)
...
+
+        /// <summary>
+        /// Send a SCI message with content with a Content Telegram Error.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public async Task SendSciMessageContentError(Empty request)
+        {
+            if (Connection == null) throw new InvalidOperationException("Connection is null. Did you call Connect()?");
+            byte[] messageBytes = new Messages.Baseline4R1.PointPointPositionMessage(
```
Comment (rs22) [inline src/Point/Point.cs]: this doesn't make a difference, but for now we kept this class free of references to a specific baseline. Should we -- just for clarity -- check the configured baseline and then construct either a Baseline4R1 or R2 message? If this message contents should change in future baselines, we would know that we have to update this portion of the code.

File: src/Point/Point.cs [hunk 2]
```diff
@@ -101,6 +103,62 @@ public async Task SendSciMessage(SciMessage message)
...
+
+        /// <summary>
+        /// Send a SCI message with content a Formal Telegram Error.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        public async Task SendSciMessageFormalError(Empty request)
+        {
+            if (Connection == null) throw new InvalidOperationException("Connection is null. Did you call Connect()?");
+            byte[] messageBytes = new Messages.Baseline4R1.PointPointPositionMessage(
```
Comment (rs22) [inline src/Point/Point.cs]: same here

File: src/Point/rasta-point-web/src/proto/PointServiceClientPb.ts [hunk 3]
```diff
@@ -6,8 +6,8 @@
 
 // Code generated by protoc-gen-grpc-web. DO NOT EDIT.
 // versions:
-// 	protoc-gen-grpc-web v1.5.0
-// 	protoc              v4.25.1
+// 	protoc-gen-grpc-web v1.4.2
```
Comment (rs22) [inline src/Point/rasta-point-web/src/proto/PointServiceClientPb.ts]: somehow I seem to use a newer version of protoc-gen-web (through brew). This leads to the diff below, where the nullable annotations are now being removed. Unfortunately, the protoc-gen-grpc-web v1.5.0 does not yet seem to be available through npm.

**Review by rs22 — APPROVED** [review]
Thanks! Minor remarks.

File: src/Point/Point.cs (hunk 1 above)
Comment (Scretch9) [reply src/Point/Point.cs]: Good point, though I feel like it's already quite messy and we should change this at some point.

**Pull request merged**